lib/philomena_proxy/scrapers/civitai.ex

```diff
@@ -6,18 +6,27 @@ defmodule PhilomenaProxy.Scrapers.Civitai do
 
   @behaviour Scraper
 
-  @url_regex ~r|\Ahttps?://(?:www\.)?civitai\.com/posts/([\d]+)/?|
+  @post_regex ~r|\Ahttps?://(?:www\.)?civitai\.com/posts/([\d]+)/?|
+  @image_regex ~r|\Ahttps?://(?:www\.)?civitai\.com/images/([\d]+)/?|
 
   @spec can_handle?(URI.t(), String.t()) :: boolean()
   def can_handle?(_uri, url) do
-    String.match?(url, @url_regex)
+    String.match?(url, @post_regex) || String.match?(url, @image_regex)
```

Comment:
Should use `or` with booleans
```suggestion
    String.match?(url, @post_regex) or String.match?(url, @image_regex)
```